CMakeLists.txt

```
@@ -46,9 +32,82 @@ set(INEXOR_APP_VERSION_MAJOR 0)
 set(INEXOR_APP_VERSION_MINOR 1)
 set(INEXOR_APP_VERSION_PATCH 0)
 
-find_package(Git REQUIRED)
+find_package(Vulkan)
+
+include(FetchContent)
+set(FETCHCONTENT_QUIET OFF)
+
+FetchContent_Declare(fmt
+    GIT_REPOSITORY https://github.com/fmtlib/fmt.git
+    GIT_TAG 8.0.1
+    GIT_SHALLOW ON
+    GIT_PROGRESS ON)
+FetchContent_MakeAvailable(fmt)
+
+set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_DOCS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+FetchContent_Declare(glfw
+    GIT_REPOSITORY https://github.com/glfw/glfw.git
+    GIT_TAG 3.3.8
+    GIT_SHALLOW ON
+    GIT_PROGRESS ON)
+FetchContent_MakeAvailable(glfw)
```

```suggestion
FetchContent_MakeAvailable(glfw FIND_PACKAGE_ARGS 3.3.8)
```

Not tested, but should ensure that we get 3.3.8. Maybe do it for the rest too? but no clue if it is `v.1.89.4` or `1.89.4` at imgui.

---

Ok I will add this.

---

I added it, but I am not sure if it's correct this way. Please check.

---

I added this for all packages except glm and stb.